plugins/scanners/scanoss/src/main/kotlin/ScanOss.kt

```diff
@@ -57,17 +56,7 @@ class ScanOss(
 
     override val configuration = ""
 
-    override val matcher by lazy {
-        ScannerMatcher.create(
-            details,
-            ScannerMatcherConfig(
-                config.regScannerName,
-                config.minVersion,
-                config.maxVersion,
-                configuration
-            )
-        )
-    }
+    override val matcher: ScannerMatcher? = null
```

Comment:
@isasmendiagus You should also change the property below to `override val readFromStorage = null` and remove the `readFromStorage` property from the config class, like it is also done in `FossId`.
I'm not requesting a change because you can also do it separately.

Reply:
I haven't seen that. I'll implement the change on PR #10287

Reply:
@isasmendiagus, please create a separate PR for this, as I want to get this in earlier than our discussion about the report generation PR will come to a conclusion.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/10291.